frontend/awx/resources/templates/WorkflowJobTemplateInputs.tsx

```diff
@@ -0,0 +1,125 @@
+        )}
+        additionalControls={
+          <PageFormCheckbox label={t('Prompt on launch')} name="ask_scm_branch_on_launch" />
+        }
+        label={t('Source control branch')}
+      />
+
+      <PageFormLabelSelect<WorkflowJobTemplateFormType>
+        labelHelpTitle={t('Labels')}
+        labelHelp={t(
+          `Optional labels that describe this job template, such as 'dev' or 'test'. Labels can be used to group and filter job templates and completed jobs.`
+        )}
+        name="summary_fields.labels.results"
+        additionalControls={
+          <PageFormCheckbox label={t('Prompt on launch')} name="ask_labels_on_launch" />
+        }
+      />
+      <PageFormSection singleColumn>
+        <PageFormDataEditor<WorkflowJobTemplateFormType>
+          additionalControls={
+            <PageFormCheckbox label={t('Prompt on launch')} name="ask_variables_on_launch" />
+          }
+          labelHelpTitle={t('Extra Variables')}
+          labelHelp={t(`Optional extra variables to be applied to job template`)}
+          toggleLanguages={['yaml', 'json']}
+          label={t('Extra Variables')}
+          name="extra_vars"
+          isExpandable
+        />
+      </PageFormSection>
+      <PageFormCreatableSelect<WorkflowJobTemplateFormType>
+        labelHelpTitle={t('Job tags')}
+        labelHelp={t(
+          'Tags are useful when you have a large playbook, and you want to run a specific part of a play or task. Use commas to separate multiple tags. Refer to the documentation for details on the usage of tags.'
+        )}
+        name="arrayedJobTags"
+        additionalControls={
+          <PageFormCheckbox label={t('Prompt on launch')} name="ask_tags_on_launch" />
+        }
+        placeholderText={t('Select or create job tags')}
+        label={t('Job tags')}
+        options={workflowJobTemplate?.arrayedJobTags ?? [{ value: '', label: '' }]}
+      />
+      <PageFormCreatableSelect<WorkflowJobTemplateFormType>
+        labelHelpTitle={t('Skip tags')}
+        labelHelp={t(
+          'Skip tags are useful when you have a large playbook, and you want to skip specific parts of a play or task. Use commas to separate multiple tags. Refer to the documentation for details on the usage of tags.'
+        )}
+        additionalControls={
+          <PageFormCheckbox label={t('Prompt on launch')} name="ask_skip_tags_on_launch" />
+        }
+        name="arrayedSkipTags"
+        placeholderText={t('Select or create skip tags')}
+        label={t('Skip tags')}
+        options={workflowJobTemplate?.arrayedSkipTags ?? [{ value: '', label: '' }]}
+      />
+      <PageFormCheckbox<WorkflowJobTemplateFormType>
+        label={t('Enable webhook')}
+        name="isWebhookEnabled"
+      />
+      <PageFormCheckbox<WorkflowJobTemplateFormType>
+        label={t('Enable Concurrent jobs')}
+        name="allow_simultaneous"
+      />
```

Comment:
Could we move these two checkboxes into a separate form section? When the form is in 3 columns, the checkboxes get split up. 
<img width="1169" alt="Screenshot 2023-09-01 at 12 07 50 PM" src="https://github.com/ansible/ansible-ui/assets/15881645/a9401bd3-3f28-4a1e-8cd1-55f76de74c2d">